TMSNet07.Homework4.Calculator/TMSNet07.Homework4.Calculator/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMSNet07.Homework4.Calculator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            char repiat = 'y';
```

Comment:
```suggestion
            const char repiat = 'y';
```
Constant values don't take memory while the application running - their values are injected into the code while compilation.